server/Idp/IDP.Domain/Models/Manager.cs

```diff
@@ -1,11 +1,14 @@
-﻿using System;
+﻿using Microsoft.EntityFrameworkCore;
+using System;
 using System.Collections.Generic;
 using System.Linq;
 using System.Text;
 using System.Threading.Tasks;
 
 namespace Idp.Domain.Models
 {
+    [Index(nameof(EmployeeId), IsUnique = true)]
+
     public class Manager
```

Comment:
The name is already there in the ApplicationUser table. Why did you add it again here?

Reply:
This one is not  login, this is different table, when removing name, it shows error. Can we use Application user for other tables than user